File: UETshare/src/main/java/com/example/uetshare/response/dto/QuestionDto.java
package com.example.uetshare.response.dto;

import com.example.uetshare.entity.Account;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Calendar;
import java.util.List;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class QuestionDto {

    private Long id;
    private String content;
    private String title;
    private Calendar time;
    private Long account_id;
    private Long type_content_id;
    private Long category_id;
    private List<ImageDto> imageDtoList;

}

File: code_fronend/app/src/main/java/ie/app/uetstudents/ui/Entity/Question/get/QuestionDto.kt
package ie.app.uetstudents.ui.Entity.Question.get

data class QuestionDto(
    val account_id: Any?,
    val categoryIdList: List<Int>?,
    val content: String,
    val id: Int?,
    val image: String?,
    val title: String,
    val type_content_id: Any?,
    val time : String?
)